standalone/src/models/associate-role/associate-role-draft/presets/empty.spec.ts

```diff
@@ -1,12 +1,52 @@
-import { TAssociateRoleDraft } from '../../types';
-import empty from './empty';
-
-it(`should set all specified fields to undefined`, () => {
-  const emptyDraft = empty().build<TAssociateRoleDraft>();
-  expect(emptyDraft).toMatchObject({
-    name: undefined,
-    buyerAssignable: undefined,
-    permissions: undefined,
-    custom: undefined,
+import type {
+  TAssociateRoleDraftRest,
+  TAssociateRoleDraftGraphql,
+} from '../../types';
+import { restPreset, graphqlPreset, compatPreset } from './empty';
+
+const validateModel = (
+  model: TAssociateRoleDraftRest | TAssociateRoleDraftGraphql
+) => {
+  expect(model).toMatchObject({
+    key: expect.any(String),
+    name: null,
+    buyerAssignable: expect.any(Boolean),
+    permissions: expect.any(Array),
```

Comment:
Should this be `expect.toBe([])`?

Reply:
Update here: 9bad310a3cf270e2d46d1c4824d6e820ff87220a